modules/ocf_www/manifests/site/shorturl.pp

```diff
@@ -42,6 +42,7 @@
       {rewrite_rule => '^/email-update$ http://status.ocf.berkeley.edu/2014/06/email-discontinuation-update-forward.html [R]'},
       {rewrite_rule => '^/facebook$ https://goo.gl/forms/dEzJmyRMwAPWCDAY2 [R]'},
       {rewrite_rule => '^/faq$ https://www.ocf.berkeley.edu/docs/faq/ [R]'},
+      {rewrite_rule => '^/gh/([^/]*)(/(?!blob)(?!tree)(?!issue)(?!pull).+)$ https://ocf.io/gh/$1/blob/master$2 [R]'},
```

Comment:
Also, it would be better to use `(/(?!blob/)(?!tree/)(?!issue/)(?!pull/).+)` or something along those lines, so we can still access a hypothetical file called `blobs/blob1`.